src/contracts/Protocol/QuestionStateController.sol

```diff
@@ -25,8 +25,8 @@ contract QuestionStateController is IQuestionStateController, Ownable, OnlyApi {
      */
     function initializeQuestion(uint256 questionId, string calldata uri) public onlyApi {
         state[questionId] = STATE.VOTING;
-        QuestionData memory _question = QuestionData({questionId: questionId, url: uri, totalVotes: getTotalVotes(questionId)});
-        questionByState[STATE.VOTING].push(_question);
+        QuestionData memory _question = QuestionData({url: uri, totalVotes: 0});
```

Comment:
when we initialize question we map it to the state of voting and the question ID so we can amend when we upvote